.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: glvd
+
+on:
+  pull_request:
+  release:
+    types:
+    - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and export
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Containerfile
+          tags: glvd:latest
```

Comment:
Where did you get this tag from?

Reply:
Should I use the hash commit or a git tag (if available) instead? 
Currently it is hardcoded, yes.

Reply:
I changed the image tag from latest to the Git commit hash or a git tag (if provided).

Reply:
You don't need any tag. The build action exports the image id.

Reply:
Ok, I see. I thought we could use the image tag to then use it for publishing afterwards. But using just the imageid is also fine.